server.go

```diff
@@ -341,6 +342,35 @@ func NewFromConfig(conf Config) (*Server, error) {
 		}
 	}
 
+	if conf.KafkaBroker != "" {
```

Comment:
Should having a Broker but no topics configured be an error?

Reply:
I added some warnings and also fleshed out the tests more thoroughly for this.